crates/wasi-nn/src/backend/onnxruntime.rs

```diff
@@ -48,7 +48,7 @@ impl BackendFromDir for OnnxBackend {
     }
 }
 
-struct ONNXGraph(Arc<Mutex<Session>>, ExecutionTarget);
+struct ONNXGraph(Arc<Mutex<Session>>, #[allow(dead_code)] ExecutionTarget);
```

Comment:
cc @devigned I had to add this as it's a new warning in rustc, and I'm not sure if this indicates a bug where this should have been used or if it should be removed.

Reply:
Thank you for the heads up. I'll take a look.